plugins/test/fu-plugin-test-invalid-build-hash.c

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (C) 2016 Richard Hughes <richard@hughsie.com>
+ *
+ * SPDX-License-Identifier: LGPL-2.1+
+ */
+
+#include "config.h"
+
+#include "fu-plugin-vfuncs.h"
+#include "fu-hash.h"
```

Comment:
Probably don't need fu-hash.h since you're just using invalid

Reply:
Agree.